GUI/SearchTaskWindow.xaml

```diff
@@ -48,15 +48,36 @@
                         <StackPanel>
                             <GroupBox Header="Deisotoping">
                                 <StackPanel>
-                                    <CheckBox x:Name="useProvidedPrecursor" Content="Use Provided Precursor" IsChecked="True" />
-                                    <CheckBox x:Name="deconvolutePrecursors" Content="Deconvolute Precursors" IsChecked="True" />
+                                    <CheckBox x:Name="useProvidedPrecursor" Content="Use Provided Precursor" IsChecked="True" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <CheckBox.ToolTip>
+
+                                            <TextBlock>
+                                                Use the charge states and precursor masses determined by the mass spectrometer.
+                                            </TextBlock>
+                                        </CheckBox.ToolTip>
+                                    </CheckBox>
+                                    <CheckBox x:Name="deconvolutePrecursors" Content="Deconvolute Precursors" IsChecked="True" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                        <CheckBox.ToolTip>
+                                            <TextBlock>
+                                                Allow MetaMorpheus to deconvolute charge states and precursor masses.
+                                                <LineBreak/>
+                                                Additionally looks for coisolation and allows for multiple peptides to be identified from a single MS2.
```

Comment:
Additionally searches for coisolated peptides, allowing for multiple peptides to be identified from a single MS2.